canonical/snapcraft.io
# Save selected territories

## Done
Stop showing modal when changing territories in settings page

## How to QA
- Go to https://snapcraft-io-4420.demos.haus/<SNAP_NAME>/settings
- Make sure "Update metadata on release" is enabled
- Make a change to the "Distribution" field
- Save the form
- You should not see a warning modal

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-2648
Fixes https://warthogs.atlassian.net/browse/WD-6609
